igor_src/CDR3SeqData.cpp

```diff
@@ -0,0 +1,50 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
```

Comment:
Could you please put up the GNU license (including your name) as for ExtractFeatures.h